front/src/components/UserMenu.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import userApi from '../api/userApi';
+import {LOGIN_URL, LOGOUT_URL, ADMIN_URL, BROWSABLE_API_URL, SWAGGER_URL } from '../api/constants';
+import IconButton from '@material-ui/core/IconButton';
+import PersonIcon from '@material-ui/icons/Person';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+
+const redirectTo = (url, next = false) => {window.location.href = url + (next ? `?next=${window.location.href}` : '')};
+
+function AdminMenuItems({ is_staff }) {
+    const toAdmin = () => redirectTo(ADMIN_URL);    
+    const toBrowsableApi = () => redirectTo(BROWSABLE_API_URL);
+    const toSwagger = () => redirectTo(SWAGGER_URL)
+    return ( is_staff ? [
+        <MenuItem key={1} onClick={toAdmin}>Django Admin</MenuItem>,
+        <MenuItem key={2} onClick={toBrowsableApi}>API Browser</MenuItem>,
+        <MenuItem key={3} onClick={toSwagger}>API Swagger</MenuItem>,
+    ] : [])
+}
+
+function UserMenu() {
+    const [anchor, setAnchor] = React.useState(false);
+    const [profile, setProfile] = React.useState(null);
+    React.useEffect(() => {
+        if (profile != null) { return; }
+        userApi.getMyProfile()
+            .then(profile => setProfile(profile))
+            .catch(() => setProfile(false));
+    })
```

Comment:
You should add this into the `store` so we can access the user's infos everywhere in the app. This way we can show different components depending on the user's status